src/dramatiq_abort/backends/redis.py

```diff
@@ -1,9 +1,13 @@
-from typing import Any, Iterable, List, Optional, Tuple
+import json
+from typing import Any, Dict, Iterable, Optional, Tuple, Union, overload
 
 import redis
 
 from ..backend import EventBackend
 
+MARKER = "dramatiq-abort_marker"
```

Comment:
to be fair, I don't think the `MARKER` is worth to keep with namespace. I would remove this.